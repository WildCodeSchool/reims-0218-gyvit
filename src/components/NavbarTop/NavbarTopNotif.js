//TODO: create component notif

import React from "react"
import PropTypes from "prop-types"
import NavbarTopNotifBubble from "./NavbarTopNotifBubble"

const NavbarTopNotif = () => {
  return (
    <div>
      <img
        style={{
          marginTop: "37px"
        }}
        src={process.env.PUBLIC_URL + "img/icon_notification.png"}
        alt="iconNotif"
      />
      <NavbarTopNotifBubble notifsCount={66} />
    </div>
  )
}

NavbarTopNotif.propTypes = {
  notifsCount: PropTypes.number.isRequired
}

export default NavbarTopNotif
