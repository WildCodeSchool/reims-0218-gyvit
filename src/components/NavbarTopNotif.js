//TODO: create component notif

import React from "react"
import { NavItem } from "reactstrap"
import NavbarTopNotifBubble from "./NavbarTopNotifBubble"

const NavbarTopNotif = ({ notifsCount }) => {
  console.log(notifsCount)
  return (
    <NavItem>
      <img src={process.env.PUBLIC_URL + "img/iconNotif.ico"} alt="iconNotif" />
      <NavbarTopNotifBubble notifsCount={notifsCount} />
    </NavItem>
  )
}

export default NavbarTopNotif
