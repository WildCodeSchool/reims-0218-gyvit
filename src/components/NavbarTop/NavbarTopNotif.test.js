//TODO: Test create component notif

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import NavbarTopNotif from "./NavbarTopNotif"

const notifsCount = 6

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<NavbarTopNotif notifsCount={notifsCount} />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer
    .create(<NavbarTopNotif notifsCount={notifsCount} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
