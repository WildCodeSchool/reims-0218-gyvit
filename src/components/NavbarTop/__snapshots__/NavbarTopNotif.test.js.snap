// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <img
    alt="iconNotif"
    src="img/icon_notification.png"
    style={
      Object {
        "marginTop": "37px",
      }
    }
  />
  <span
    className="badge badge-secondary badge-pill"
    style={
      Object {
        "backgroundColor": "#ff007e",
        "height": "20px",
        "marginTop": "-50px",
        "width": "auto",
      }
    }
  >
    8
  </span>
</div>
`;
